app/api/__init__.py

```diff
@@ -398,3 +401,7 @@
 api.route(OrderRelationship, 'order_event', '/orders/<identifier>/relationships/event')
 api.route(OrderRelationship, 'order_marketer', '/orders/<identifier>/relationships/marketer')
 api.route(OrderRelationship, 'order_discount', '/orders/<identifier>/relationships/discount-code')
+
+# Event Analytics API
+api.route(EventAnalyticsGeneralDetail, 'event_analytics_general_detail', '/analytics/events/<int:id>/general',
```

Comment:
I did that initially if you see the PR email,  but hen as https://github.com/fossasia/open-event-orga-server/issues/4239 came up, I changed it to `analytics/events/{event_identifier}/general` so that all such endpoints fall under a common `/analytics` prefix. What do you think :) ?